runtime/wasm/src/module/mod.rs

```diff
@@ -793,6 +798,21 @@ where
             .big_decimal_equals(self.asc_get(x_ptr), self.asc_get(y_ptr));
         Ok(Some(RuntimeValue::I32(if equals { 1 } else { 0 })))
     }
+
+    /// function dataSource.create(name: string, params: Array<string>): void
```

Comment:
The comment says this returns void but we're returning something in the function.

Reply:
Changed to _actually_ return `void` now.